ui/src/components/gallery/QasGallery.vue

```diff
@@ -39,96 +41,89 @@ export default {
       default: 'o_chevron_right'
     },
 
-    carouselPrevIcon: {
+    carouselPreviousIcon: {
       type: String,
       default: 'o_chevron_left'
     },
 
-    imageHeight: {
+    height: {
       type: String,
       default: ''
     },
 
-    initialPhotosQuantity: {
+    initialLength: {
       type: Number,
       default: 6
     },
 
-    labelShowMore: {
-      type: String,
-      default: 'Ver mais'
-    },
-
-    photos: {
+    items: {
       type: Array,
-      default: () => ([])
+      default: () => []
     },
 
-    photosQuantityToLoad: {
+    loadLength: {
       type: Number,
       default: 6
+    },
+
+    showMoreLabel: {
+      type: String,
+      default: 'Ver mais'
     }
   },
 
   data () {
     return {
       carouselDialog: false,
-      photoIndex: [],
-      displayedPhotos: this.initialPhotosQuantity
+      itemIndex: [],
+      displayeditems: this.initialLength
     }
   },
 
   computed: {
-    initialPhotos () {
-      return this.photos.slice(0, this.displayedPhotos)
+    initialitems () {
+      return this.items.slice(0, this.displayeditems)
     },
 
-    photoHeight () {
-      if (this.isSingleImage && !this.imageHeight) return 'auto'
-
-      if (this.isSingleImage && this.imageHeight) return this.imageHeight
+    itemHeight () {
+      if(this.isSingleItem) {
```

Comment:
`return this.height || 'auto'` não resolve? e esta faltando espaço no if

Reply:
corrigido